syncTree.sh

```diff
@@ -6,9 +6,10 @@ REVISION="feaa9eace7098c343598bf08fb50746a1e8d2deb"
 
 echo "tree service revision ${REVISION}"
 
-FILES=$(curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | sed -n "s,.*\"/nspcc-dev/neofs-node/blob/${REVISION}/pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p")
+FILES=$(curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | jq | sed -n "s,.*\"pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p")
```

Comment:
What's the effect?

Reply:
Without these changes syncTree doesn't work - it doesn't find any files in _REVISION_.
I tend to think, GH changed the response format a little bit:
* shorter files pass, removed `/nspcc-dev/neofs-node/blob/${REVISION}` from file path
* and JSON is not formatted - just one string.

Unfortunately, I don't know how it was before, but without `jq` sed finds only one file. With formatted JSON - all files. How I understand it checks input by each row

Reply:
```shell
$ REVISION="feaa9eace7098c343598bf08fb50746a1e8d2deb"

Without jq
$ curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | sed -n "s,.*\"pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p"
types_neofs.pb.go

With jq
$ curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | jq | sed -n "s,.*\"pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p"
service.pb.go
service_grpc.pb.go
service_neofs.pb.go
types.pb.go
types_neofs.pb.go